.github/workflows/update-errorcodes.yml

```diff
@@ -0,0 +1,39 @@
+name: Create PR for new error codes
+
+on: [workflow_dispatch]
```

Comment:
This would be triggered from the release workflows over in @apollo/client.

Reply:
We're currently one release behind with this PR, which will give us a great opportunity to test this workflow :)